.github/workflows/doxygen-gh-pages.yml

```diff
@@ -0,0 +1,48 @@
+name: Doxygen GitHub Pages Deploy Action
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
```suggestion
```

Remove the `on: pull_request` option once the PR is ready. This option is only used for debugging so that the Action runs on this PR.